elyra/scheduler/handler.py

```diff
@@ -42,10 +42,13 @@ def post(self, *args, **kwargs):
             json_msg = json.dumps({"status": "ok",
                                    "message": "Pipeline successfully exported"})
         else:
-            run_url = PipelineProcessorManager.process(pipeline)
+            response = PipelineProcessorManager.process(pipeline)
             json_msg = json.dumps({"status": "ok",
                                    "message": "Pipeline successfully submitted",
-                                   "url": run_url})
+                                   "experiment-url": response.experiment_url,
+                                   "object-storage-url": response.object_storage_url,
+                                   "object-storage-path": response.object_storage_path
+                                   })
```

Comment:
I feel like this response is adding too much about _presentation_.  Why is `status` and `message` part of the response payload at all?  Status should derive from the status code and message should derive from the endpoint.

This should also be described in the [swagger yaml](https://github.com/elyra-ai/elyra/blob/master/elyra/api/elyra.yaml) once settled.

Reply:
In this case I agree with @kevin-bates , when the return json has expected return values only those values should be in the response.